### ISSUE TITLE
Check for Updates Popup Buttons are Frozen Temporarily

### ISSUE THREAD

**Issue by JamesGeiger** [MonitorControl/MonitorControl#1665]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When I go to Check for Updates I can't click the buttons for a few seconds - not a major issue but annoying.

<img width="278" alt="Screenshot 2024-11-04 at 6 11 02 PM" src="https://github.com/user-attachments/assets/4fa1b6af-e334-4836-864d-2fa3c7116d81">

### Steps to reproduce

1. Open App
2. Click on Check for Updates
3. Try clicking on OK or Version History and they don't work for a few seconds

### Expected behavior

Expected behavior would be you can click buttons immediately 

### Environment Information (please complete the following information)

```markdown
- macOS version: 15.1
- Mac model: M1 MacBook Air
- MonitorControl version: 4.3.3
- Monitor(s): 1 LG UltraFine 4k (27 Inch)
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Yes, this is how Sparkle (the updater component) seems to work. Since it's a [third party project](https://github.com/sparkle-project/Sparkle), I am not sure I can fix this at my end. But I noticed the behavior too (both with MonitorControl and other apps I do like BetterDisplay that uses sparkle).
